emukit/core/initial_designs/sobol_design.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
```

Comment:
licence header missing

Reply:
Thanks, added. Side note, the year should probably be updated across the lib.